packages/lodestar/src/api/impl/beacon/state/errors.ts

```diff
@@ -0,0 +1,7 @@
+import {ApiError} from "../../errors/api";
+
+export class MissingState extends ApiError {
```

Comment:
how is this different from `StateNotFound` error?

Reply:
It isn't, gonna remove this one